editor/store/effects.js

```diff
@@ -323,34 +323,36 @@ export default {
 		const { dispatch } = store;
 
 		let result;
-		if ( id ) {
+		if ( id && wp.api.models.Blocks ) {
```

Comment:
Since this is a temporary fix, it's probably worth adding a `// TODO: ` comment above the checks.